Source/GTMSessionFetcher.h

```diff
@@ -633,10 +593,6 @@ NSData *_Nullable GTMDataFromInputStream(NSInputStream *inputStream, NSError **o
 }  // extern "C"
 #endif
 
-#if !GTM_USE_SESSION_FETCHER
```

Comment:
I _think_ this and the conditional definition on the `fetcherService` property is likely safe to since the on things that seem like they could break would be someone explicitly using the SessionFetcher but with the `#define` for the bridge macros set to `0`, and that seems like it an odd state.

Reply:
Agreed. I don't think setting GTM_USE_SESSION_FETCHER=0 is really a viable configuration at this point.